debugging.rst

```diff
@@ -0,0 +1,84 @@
+.. _debugging:
+
+==============
+Debugging Jobs 
+==============
+
+Flux supports parallel debuggers such as Rogue Wave Software (RWS)'s
+TotalView parallel debugger (https://www.roguewave.com).
+
+----------------------------------
+Parallel Debugging using TotalView
+----------------------------------
+
+Debugging your MPI job from the beginning of
+parallel program execution:
+
+.. code-block:: sh
+
+  totalview --args flux mini run -N 2 -n 2 ./mpi-program
+
+Attaching to an already running job:
+
+.. code-block:: sh
+
+  totalview --args flux mini attach <JOBID>
+
+Note that you should use ``totalview --args flux mini attach`` whether your
+job has been launched via ``flux mini run`` or ``flux mini submit``.
+In other word, Flux currently does not allow TotalView
+to attach to your running code if you just attach it
+to the ``flux mini run`` process that controls your code.
+
+Flux provides ``MPIR_partial_attach_ok`` (https://www.mpi-forum.org/docs/mpir-specification-10-11-2010.pdf)
```

Comment:
I **think** it is possible to combine the verbatim mode (``) with hyperlinks, but I don't want to suggest something without trying it first.  If it isn't easily doable, maybe something like:

```suggestion
Flux provides ``MPIR_partial_attach_ok`` `[1]<https://www.mpi-forum.org/docs/mpir-specification-10-11-2010.pdf>`_
```

Reply:
OK. Will try that.